app/src/main/java/com/karumi/ui/Lifecycle.kt

```diff
@@ -3,6 +3,7 @@ package com.karumi.ui
 val lifeCycleLinker = LifecycleLinker()
```

Comment:
Now that I'm seeing this... I'm seeing a `registerToLifecycle` method but no `unregister` method. Besides, I'm seeing a problem with this being a global variable... storing strong references to presenters and not ever cleaning them up 🤔